distribution.md

```diff
@@ -69,3 +70,53 @@ Each stack ID MUST only be present once.
 For a given stack, the `mixins` list MUST enumerate mixins such that no included buildpacks are missing a mixin for the stack.
 
 Fewer stack entries as well as additional mixins for a stack entry MAY be specified.
+
+### Lifecycle
+
+The following defines how a `lifecycle` SHOULD be packaged for distribution as an OCI image. The `lifecycle` is the component that orchestrates buildpack execution, then assembles the resulting artifacts into a final app image.
+
+The Lifecycle MUST exist in one of the following formats:
+
+* An OCI Image on an image registry.
+* An OCI Image in a Docker daemon.
+* An uncompressed tar archive in [oci-layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format.
+
+#### Filesystem
+
+A lifecycle image MUST have the following directories/files
+
+- `/cnb/lifecycle/<lifecycle binaries>` &rarr; An implementation of the lifecycle, which contains the required lifecycle binaries for [building images](https://github.com/buildpacks/spec/blob/main/platform.md#build).
+
+#### Labels
+
+| Label             | Description
+| --------          | --------
+| `io.buildpacks.lifecycle.version`  | A string, representing the semver version of the lifecycle.
+| `io.buildpacks.lifecycle.apis`     | A JSON object representing the APIs the lifecycle supports.
+
+`io.buildpacks.lifecycle.apis` (JSON)
+
+```json
+{
+  "buildpack": {
+    "deprecated": ["<list of versions>"],
+    "supported": ["<list of versions>"]
+  },
+  "platform": {
+    "deprecated": ["<list of versions>"],
+    "supported": ["<list of versions>"]
+  }
+}
+```
+Where:
+
+* `supported`:
+    * contains an array of support API versions:
+    * for versions `1.0+`, version `x.n` implies support for [`x.0`, `x.n`]
+    * should be a superset of `deprecated`
+    * should only contain APIs that correspond to a spec release
+
+* `deprecated`:
+    * contain an array of deprecated APIs:
+    * should only contain `0.x` or major versions
+    * should only contain APIs that correspond to a spec release
```

Comment:
I believe these require indentation since they should be a sub-list of "contain an array of deprecated APIs:"